compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,133 @@
+; To activate a step (or task),
+; set True for the step in [onecc] section and fill options in the corresponding section
 [onecc]
-one-import-tf=True
+; neural network model to circle
+one-import-tf=False
 one-import-tflite=False
 one-import-bcq=False
 one-import-onnx=False
-one-optimize=True
+; circle to circle with optimization
+one-optimize=False
+; circle to circle with quantization
 one-quantize=False
+; partition circle
 one-partition=False
-one-pack=True
+; package circle and metadata into nnpackage
+one-pack=False
+; generate code for backend
 one-codegen=False
+; profile
 one-profile=False
 
 [one-import-tf]
-input_path=/path/to/inception_v3.pb
-output_path=inception_v3.circle
-input_arrays=input
-input_shapes=1,299,299,3
-output_arrays=InceptionV3/Predictions/Reshape_1
-converter_version=v1
-model_format=graph_def
+# mandatory
+; pb file
+input_path=
+; circle file
+output_path=
+# optional
+; v1 or v2
+converter_version=v2
+# optional but mandatory for converter_version=v1
```

Comment:
@YongseopKim , I am sorry to give you incorrect information.
input_arrays and other options are needed for if model format is `graph_def`, which is default.
others are `saved_model` and `keras_model`.

Reply:
Plz refer `compiler/tf2tfliteV2/tf2tfliteV2.py` file for details.

Reply:
Thanks! I fix